### ISSUE TITLE
Bug: No yank highlight when using `Y` (remaps)

### ISSUE THREAD

**Issue by chrisgrieser** [aleksey-rowan/obsidian-vim-yank-highlight#3]
Using the vimrc plugin, I have: `nnoremap Y y$`

When I yank something with `y`, the highlight appears just fine. When I yank with `Y`, no highlight appears.

**Comment by aleksey-rowan** [comment]
That makes sense, thank you for reporting. Codemirror-vim doesn't expose yank events (and I don't want to monkey-patch it), so I have to infer when a yank happens.

I have a couple of bad ideas on how to fix this.

**Comment by chrisgrieser** [comment]
A simple solution could be to add a setting where the user can name extra yank characters like Y?

**Comment by aleksey-rowan** [comment]
Yes, adding a user setting will help and I though this could be a potential solution, but I don't think it's optimal as it adds more friction and forces the user to tinker with the settings. Also, if I add highlight on put/undo/redo later (which I want to do), I would need to add more settings in case any of these commands are also remapped.

As a temporary fix (and we all know how permanent they are), I added `Y` to highlight triggers which is only proper since `Y` yanks the whole line by default anyway. Check [1.0.8](https://github.com/aleksey-rowan/obsidian-vim-yank-highlight/releases/tag/1.0.8).

For a proper solution, I would like to snoop at the mappings if they are exposed by codemirror-vim and try to figure out when a remapped yank is called. If that fails, I'll probably go with the user setting approach.

**Comment by Dzhegash** [comment]
Yes, indeed, it is very necessary to configure additional options for the selection keys, for example, I use map for "yiw" in Cyrillic it is "ншц" and with such a selection, the backlight does not work.
